test/libdnf/utils.hpp

```diff
@@ -68,5 +68,8 @@ std::vector<std::string> to_vector(const libdnf::rpm::ReldepList & rdl);
 /// Convert PackageSet to a vector of strings for easy assertions.
 std::vector<std::string> to_vector(const libdnf::rpm::PackageSet & pset);
 
+/// Convert vector<Package> to a vector of strings for easy assertions.
+std::vector<std::string> to_vector(const std::vector<libdnf::rpm::Package> & pkg_list);
```

Comment:
I'd say name this in a bit more specific way, it already takes a vector as an input. `package_to_nevra_vector` maybe?

Reply:
I used `to_string` to make it consistent with the other functions that do the same.
In case we rename it, shouldn't we rename all of them?

Reply:
Well the other two functions present here convert custom container classes to vectors, so there the name makes sense.

`to_string` doesn't seem great either, as it is actually a vector of strings (specifically NEVRAs).

(This is just about a name of utility function in tests, so of very minor importance, let's just use something that's not too much off :slightly_smiling_face: )